src/main/java/edu/kis/powp/jobs2d/features/MouseDrawFeature.java

```diff
@@ -0,0 +1,32 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import javax.swing.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+public class MouseDrawFeature {
+    private static final int Width = 268;
```

Comment:
CONSTANTS are written uppercase. What width and what kind? Relative (delta), absolute?